Tests/Integration/Components/ImageImportTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ShopwarePlugins\Connect\Tests\Integration\Components;
+
+use ShopwarePlugins\Connect\Components\Logger;
+use Shopware\Models\Article\Supplier;
+use Shopware\Models\Article\Image;
+use Shopware\Models\Media\Media;
+use ShopwarePlugins\Connect\Components\ImageImport;
+use ShopwarePlugins\Connect\Tests\ConnectTestHelperTrait;
+use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
+
+class ImageImportTest extends \PHPUnit_Framework_TestCase
+{
+    use DatabaseTestCaseTrait;
+    use ConnectTestHelperTrait;
+
+    private $manager;
+    /** @var ImageImport */
+    private $imageImport;
+
+    public function SetUp()
+    {
+        $this->manager = Shopware()->Models();
+        $this->imageImport = new ImageImport(
+            Shopware()->Models(),
+            Shopware()->Plugins()->Backend()->SwagConnect()->getHelper(),
```

Comment:
yes but after using the productBuilderTrait this would be the only usage of the connectTestHelperTrait 
i decided to use this and not depend on the ConnectTestHelperTrait at all